### ISSUE TITLE
Replace Ruby Sass

### ISSUE THREAD

**Issue by jendiamond** [projectblacklight/blacklight#2093]
[Ruby Sass should no longer be used, and will no longer be receiving any updates. See the Sass blog, and consider switching to the sassc gem.](https://github.com/sass/ruby-sass)

---

### [Ruby Sass Has Reached End-Of-Life](http://sass.logdown.com/posts/7828841)

One year has passed since we announced the deprecation of Ruby Sass, and it has now officially reached its end-of-life. We will release one final version of the Ruby Sass gem that will print a warning indicating that it's no longer receiving updates, and then archive the GitHub repository.

We will then merge the `sass/language` repo into the `sass/sass` repo. This means that anyone still depending on Ruby Sass from github.com/sass/sass will break. Going forward, the `sass/sass` repo will be the location for working on the language specs, and will not contain any code. The sass/language repo will just include links pointing to `sass/sass`.

### Migrating Away
If you haven't migrated away from Ruby Sass yet, now is the time. The best way to do that depends on how you use Ruby Sass today...

#### If you're using Rails, we particularly recommend using the [**`sassc-rails gem`**,](https://github.com/sass/sassc-rails) which wraps up the sassc gem and integrates it smoothly into the asset pipeline. Most of the time you won't even need to change any of your code.

**Comment by jcoyne** [comment]
I don't believe that Blacklight has a dependency on sass.  Can you point where you're seeing this?

**Comment by jendiamond** [comment]
`Gemfile` Line #39 && #43

<img width="348" alt="Screen Shot 2019-05-01 at 11 19 28 AM" src="https://user-images.githubusercontent.com/751697/57033926-11bfe080-6c03-11e9-9704-2be100a3c438.png">


They can be replaced with `gem 'sassc-rails'`

**Comment by jcoyne** [comment]
Since we don't support rails < 5  anymore, we can probably just remove those clauses.  I believe those were generated by the `engine-cart` gem.
